core/arch/arm/plat-imx/registers/imx6-crm_regs.h

```diff
@@ -0,0 +1,861 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2017-2018 NXP
+ *
+ */
+#ifndef __MX6_CCM_REGS_H__
+#define __MX6_CCM_REGS_H__
+
+#define CCM_CCR			(0x0000)
+#define CCM_CCDR		(0x0004)
+#define CCM_CSR			(0x0008)
+#define CCM_CCSR		(0x000C)
+#define CCM_CACRR		(0x0010)
+#define CCM_CBCDR		(0x0014)
+#define CCM_CBCMR		(0x0018)
+#define CCM_CSCMR1		(0x001C)
+#define CCM_CSCMR2		(0x0020)
+#define CCM_CSCDR1		(0x0024)
+#define CCM_CS1CDR		(0x0028)
+#define CCM_CS2CDR		(0x002C)
+#define CCM_CDCDR		(0x0030)
+#define CCM_CHSCCDR		(0x0034)
+#define CCM_CSCDR2		(0x0038)
+#define CCM_CSCDR3		(0x003C)
+#define CCM_CSCDR4		(0x0040)
+#define CCM_CWDR		(0x0044)
+#define CCM_CDHIPR		(0x0048)
+#define CCM_CDCR		(0x004C)
+#define CCM_CTOR		(0x0050)
+#define CCM_CLPCR		(0x0054)
+#define CCM_CISR		(0x0058)
+#define CCM_CIMR		(0x005C)
+#define CCM_CCOSR		(0x0060)
+#define CCM_CGPR		(0x0064)
+#define CCM_CCGR0		(0x0068)
+#define CCM_CCGR1		(0x006C)
+#define CCM_CCGR2		(0x0070)
+#define CCM_CCGR3		(0x0074)
+#define CCM_CCGR4		(0x0078)
+#define CCM_CCGR5		(0x007C)
+#define CCM_CCGR6		(0x0080)
+#define CCM_CCGR7		(0x0084)
+#define CCM_CMEOR		(0x0088)
```

Comment:
'()' is not good. Are these registered used somewhere? Please do not add things not used now.